Runtime/MeshOperations/AppendElements.cs

```diff
@@ -229,6 +229,20 @@ internal static ActionResult CreateShapeFromPolygon(this PolyShape poly)
         /// <param name="flipNormals">If true the faces will be inverted at creation.</param>
         /// <returns>An ActionResult with the status of the operation.</returns>
         public static ActionResult CreateShapeFromPolygon(this ProBuilderMesh mesh, IList<Vector3> points, float extrude, bool flipNormals)
```

Comment:
this is maybe a style preference, but why not make Vector3.up the default value for cameraLookAt?

Reply:
Given that it is a public API it needs to be unchanged as far as I understood.